.cookiecutter.json

```diff
@@ -1,26 +1,35 @@
 {
   "cookiecutter": {
-    "codeowner_github_usernames": "@smith-ntc",
+    "codeowner_github_usernames": "@brohrich @joewesch @jvanderaa @LoneStar-Swish @nniehoff @smk4664 @glennmatthews @chadell",
     "full_name": "Network to Code, LLC",
     "email": "info@networktocode.com",
     "github_org": "nautobot",
-    "plugin_name": "nautobot_welcome_wizard",
-    "verbose_name": "Nautobot's Welcome Wizard",
+    "plugin_name": "welcome_wizard",
+    "verbose_name": "Welcome Wizard",
     "plugin_slug": "nautobot-welcome-wizard",
     "project_slug": "nautobot-plugin-welcome-wizard",
     "repo_url": "https://github.com/nautobot/nautobot-plugin-welcome-wizard",
-    "base_url": "welcome-wizard",
-    "min_nautobot_version": "1.5.4",
-    "max_nautobot_version": "1.9999",
-    "nautobot_version": "stable",
+    "base_url": "nautobot-welcome-wizard",
+    "min_nautobot_version": "2.0.0",
+    "max_nautobot_version": "2.9999",
```

Comment:
I believe that the version number here would not make it work with 1.6?

Reply:
Needed to merge this prior to merging the 2.0 work, need to checkout commit and redo PR